<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>650</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>开  始  识  别</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>730</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>结  束  识  别</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>570</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>选  择  文  件</string>
    </property>
   </widget>
   <widget class="QTableView" name="OutputView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>570</y>
      <width>800</width>
      <height>220</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DisplayLabel">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>20</y>
      <width>950</width>
      <height>520</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
